Makefile

```diff
@@ -8,6 +8,9 @@ KEEP_DB ?= 0
 # TESTS: Specify individual test files in a space separated lists. As the user
 # most likely wants only the mentioned tests to be executed and no other
 # checks this implicitly disables CHECKSTYLE
+# CONTAINER_TEST: Set to 0 to exclude container tests needing a container
+# runtime environment
+CONTAINER_TEST ?= 1
```

Comment:
Since you're adding this flag I'm thinking a message like "Set CONTAINER_TEST=0 to skip container checks" would be good to have. Otherwise it won't be easy to find. And this would be similar to other flags we have for heavy tests etc.

Reply:
CHECKSTYLE is different since you don't need to set it. You can use the targets without knowing about it

Reply:
Yes, sure. I selected this approach for CONTAINER_TEST with the default to be enabled because `make test` would still try to run all tests, including the container checks. We only exclude that from circleCI because we explicitly wanted to run more in GHA and not all in circleCI.